sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates_async.py

```diff
@@ -194,7 +196,8 @@ class TestExamplesKeyVault(AsyncKeyVaultTestCase):
         )
 
         cert_name = "cert-name"
-        create_certificate_poller = certificate_client.create_certificate(name=cert_name, policy=cert_policy)
+        polling_interval = 0 if self.is_playback() else 5
```

Comment:
Could use `None` when live to get the method's default.

Reply:
forgot to switch those in this file, thanks!